.github/workflows/release_ci.yml

```diff
@@ -64,14 +64,10 @@ jobs:
         password: ${{ secrets.DOCKERHUB_PASSWORD }}
     - name: Deploy preview docker image for linux
       if: matrix.os == 'ubuntu-latest'
-      # Buildkit Only working in linux
-      env:
-        DOCKER_BUILDKIT: 1
```

Comment:
This should normally stay there unless you want also to remove the Dockerfile-CI.dockerignore file.

Reply:
I was not sure if it was being used as we were copying the build files a couple lines below.

Reply:
Dockerfile-CI doesn't need to use the DOCKER_BUILDKIT env var. This env var says to Docker itself to enable it's Docker Buildkit feature which allows to use alternate dockerignore files.